test/input/yaml/CustomInput_routelink_csv_alternative.yaml

```diff
@@ -0,0 +1,82 @@
+  #initial input parameters
+  run_parameters:
+      parallel_compute_method: 'by-subnetwork-jit-clustered'
+      subnetwork_target_size: 150
+      cpu_pool: 12
+      verbose: true
+      showtiming: true
+      debuglevel: 1
+      break_network_at_waterbodies: true
+      assume_short_ts: true
+      qts_subdivisions: 12
+      dt: 300
+      nts: 1440
+  output_parameters:
+      #output location for csv file
+      csv_output:
+          csv_output_folder: "../../test/output/text"
+          csv_output_segments: [4185713, 2743396, 4153198, 4186293, 4186169]
```

Comment:
- [ ] New issue/New Pull Request "Add segment filtering to CSV output"
    - [ ] Change these segment IDs to something relevant for the domain to be tested (if we use "CustomInput.yaml, they should already work...)
    - [ ] Test placing a filter on the csv output to limit data to the specified IDs
    - [ ] Make sure that if `csv_output_segments` is missing, the output is the full set.
    - [ ] If `csv_output_segments` is `[]` output an empty csv (with the header) for testing.
    - [ ] if `csv_output` is missing from the .yaml, make sure it doesn't crash